src/components/popups/InspectTask.tsx

```diff
@@ -192,60 +146,24 @@ const InspectTask = ({ isOpen, onClose, task, inSession }) => {
   return (
     <div className="inspectTask overlay" onClick={doClose}>
       <div className="inspectTask content" onClick={(e) => e.stopPropagation()}>
-        <div className="inspectTask header">
-          {!editMode && (
-            <IconButton
-              hoverIcon={IoMdCloseCircle}
-              icon={IoMdCloseCircleOutline}
-              onClick={doClose}
-              className="red-icon"
-              style={{ scale: "2.5", marginLeft: "10px", marginTop: "5px" }}
-            />
-          )}
-          {editMode && (
-            <>
-              <div></div>
-              <IconButton
-                hoverIcon={MdEditOff}
-                icon={MdOutlineEditOff}
-                onClick={DeactivateEditMode}
-                className="red-icon"
-                style={{ scale: "2.5", marginRight: "10px", marginTop: "5px" }}
-              />
-            </>
-          )}
-          {!editMode && !inSession && (
-            <IconButton
-              hoverIcon={MdModeEditOutline}
-              icon={MdOutlineModeEdit}
-              onClick={ActivateEditMode}
-              className="green-icon"
-              style={{ scale: "2.5", marginRight: "10px", marginTop: "5px" }}
-            />
-          )}
-        </div>
-        <h3 className="inspectTask headline">Task Title</h3>
+        <PopupHeader onClose={onClose} title="Edit Task"></PopupHeader>
```

Comment:
It should only say "Edit Task" when we are in Edit Mode

Reply:
Has been fixed, thanks a lot for your help!